src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    protected static final String type = "rectangle";
```

Comment:
The 'type' field should not be static. Each instance of 'Rectangle' should have its own 'type' field if it's necessary, or it should be removed if the type is always 'rectangle'.